File: vdp-c-sharp/vdp-c-sharp/VDPApiTest/vta/ManageCommunitiesTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Configuration;
using System.Collections.Generic;

namespace Vdp
{
    [TestClass]
    public class ManageCommunitiesTest : AbstractClient
    {
        public ManageCommunitiesTest()
        {
            
        }

        [TestMethod]
        public void TestGetCommunities()
        {
            string baseUri = "vta/";
            string resourcePath = "v3/communities";
            Dictionary<string, string> headers = new Dictionary<string, string>();
            headers.Add("ServiceId", ConfigurationManager.AppSettings["vtaServiceId"]);
            string status = DoMutualAuthCall(baseUri + resourcePath, "GET", "Get Communities Test", "", headers);
            Assert.AreEqual(status, "OK");
        }
    }
}

File: vdp-php/tests/vta/ManageCommunitiesTest.php
<?php

namespace Vdp;

class ManageCommunitiesTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->conf = parse_ini_file ( "configuration.ini", true );
		$this->abstractVisaAPIClient = new AbstractVisaAPIClient;
	}
	
	public function testGetCommunities() {
		$baseUrl = "vta/";
		$resourcePath = "v3/communities";
		$statusCode = $this->abstractVisaAPIClient->doMutualAuthCall( 'get', $baseUrl.$resourcePath, 'Get Communities Test', '', array("ServiceId: ".$this->conf['VDP'] ['vtaServiceId'] ));
		$this->assertEquals($statusCode, "200");
	}
}